<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Modifier</class>
 <widget class="QDialog" name="Modifier">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>811</width>
    <height>631</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="Line" name="line_29">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>330</y>
     <width>3</width>
     <height>61</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>300</y>
     <width>241</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Bahnschrift</family>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Date de naissance</string>
   </property>
  </widget>
  <widget class="Line" name="line_4">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>130</y>
     <width>20</width>
     <height>61</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QLineEdit" name="dateEdit">
   <property name="geometry">
    <rect>
     <x>62</x>
     <y>341</y>
     <width>231</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
  </widget>
  <widget class="Line" name="line_38">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>40</y>
     <width>341</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>400</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Bahnschrift</family>
     <pointsize>24</pointsize>
    </font>
   </property>
   <property name="text">
    <string>CIN</string>
   </property>
  </widget>
  <widget class="Line" name="line_39">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>600</y>
     <width>551</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLineEdit" name="adresseEdit">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>540</y>
     <width>231</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLineEdit" name="SexeEdit">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>450</y>
     <width>113</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>300</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Bahnschrift</family>
     <pointsize>24</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Lieu</string>
   </property>
  </widget>
  <widget class="Line" name="line_12">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>330</y>
     <width>20</width>
     <height>61</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="Line" name="line_7">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>230</y>
     <width>20</width>
     <height>61</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="Line" name="line_26">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>320</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line_6">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>220</y>
     <width>131</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line_17">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>430</y>
     <width>20</width>
     <height>61</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="Line" name="line_23">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>520</y>
     <width>21</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line_24">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>530</y>
     <width>3</width>
     <height>61</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="Line" name="line_30">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>380</y>
     <width>201</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>180</y>
     <width>291</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line_16">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>420</y>
     <width>191</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>180</y>
     <width>131</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Bahnschrift</family>
     <pointsize>24</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Pr√©nom</string>
   </property>
  </widget>
  <widget class="Line" name="line_28">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>320</y>
     <width>21</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line_36">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>20</width>
     <height>551</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="Line" name="line_10">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>280</y>
     <width>291</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>190</y>
     <width>211</width>
     <height>111</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Bahnschrift</family>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="title">
    <string>Sexe</string>
   </property>
   <widget class="QRadioButton" name="radioMale">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Bahnschrift</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Male</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioFemelle">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>50</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Bahnschrift</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Femelle</string>
    </property>
   </widget>
  </widget>
  <widget class="Line" name="line_14">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>330</y>
     <width>3</width>
     <height>61</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="Line" name="line_3">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>130</y>
     <width>3</width>
     <height>61</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>811</width>
     <height>631</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="CINEdit">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>440</y>
     <width>231</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>340</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="Line" name="line_13">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>320</y>
     <width>21</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLineEdit" name="LieuEdit">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>340</y>
     <width>171</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>120</y>
     <width>191</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line_21">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>520</y>
     <width>131</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line_37">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>50</y>
     <width>20</width>
     <height>561</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="Line" name="line_5">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>120</y>
     <width>21</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line_15">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>380</y>
     <width>291</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line_25">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>580</y>
     <width>291</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line_22">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>530</y>
     <width>20</width>
     <height>61</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>490</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Bahnschrift</family>
     <pointsize>24</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Adresse</string>
   </property>
  </widget>
  <widget class="Line" name="line_19">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>430</y>
     <width>3</width>
     <height>61</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>100</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Bahnschrift</family>
     <pointsize>24</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Nom</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>340</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>6</pointsize>
    </font>
   </property>
   <property name="text">
    <string>     Vider les champs</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>270</y>
     <width>61</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>61</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="Line" name="line_8">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>220</y>
     <width>21</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>270</y>
     <width>171</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Bahnschrift</family>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>     Confirmer</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="nomEdit">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>140</y>
     <width>231</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
  </widget>
  <widget class="Line" name="line_20">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>480</y>
     <width>291</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLineEdit" name="prenomEdit">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>240</y>
     <width>231</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
  </widget>
  <widget class="Line" name="line_18">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>420</y>
     <width>31</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>20</y>
     <width>191</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>Modifier</string>
   </property>
  </widget>
  <widget class="Line" name="line_11">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>320</y>
     <width>31</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line_27">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>330</y>
     <width>20</width>
     <height>61</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="Line" name="line_9">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>230</y>
     <width>3</width>
     <height>61</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <zorder>SexeEdit</zorder>
  <zorder>label_2</zorder>
  <zorder>line_29</zorder>
  <zorder>label_5</zorder>
  <zorder>line_4</zorder>
  <zorder>dateEdit</zorder>
  <zorder>line_38</zorder>
  <zorder>label_6</zorder>
  <zorder>line_39</zorder>
  <zorder>adresseEdit</zorder>
  <zorder>label_8</zorder>
  <zorder>line_12</zorder>
  <zorder>line_7</zorder>
  <zorder>line_26</zorder>
  <zorder>line_6</zorder>
  <zorder>line_17</zorder>
  <zorder>line_23</zorder>
  <zorder>line_24</zorder>
  <zorder>line_30</zorder>
  <zorder>line_2</zorder>
  <zorder>line_16</zorder>
  <zorder>label_4</zorder>
  <zorder>line_28</zorder>
  <zorder>line_36</zorder>
  <zorder>line_10</zorder>
  <zorder>groupBox</zorder>
  <zorder>line_14</zorder>
  <zorder>line_3</zorder>
  <zorder>CINEdit</zorder>
  <zorder>line_13</zorder>
  <zorder>LieuEdit</zorder>
  <zorder>line</zorder>
  <zorder>line_21</zorder>
  <zorder>line_37</zorder>
  <zorder>line_5</zorder>
  <zorder>line_15</zorder>
  <zorder>line_25</zorder>
  <zorder>line_22</zorder>
  <zorder>label_7</zorder>
  <zorder>line_19</zorder>
  <zorder>label_3</zorder>
  <zorder>pushButton_3</zorder>
  <zorder>line_8</zorder>
  <zorder>pushButton_2</zorder>
  <zorder>nomEdit</zorder>
  <zorder>line_20</zorder>
  <zorder>prenomEdit</zorder>
  <zorder>line_18</zorder>
  <zorder>pushButton</zorder>
  <zorder>line_11</zorder>
  <zorder>line_27</zorder>
  <zorder>line_9</zorder>
  <zorder>label</zorder>
  <zorder>label_10</zorder>
  <zorder>label_11</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
